### PR TITLE
update ID lang

### PR DESCRIPTION
**Pull request by bagoesz21**
update ID language

### CODE REVIEW

File: locales/id/validation-inline.php [hunk 1]
```diff
@@ -90,15 +90,15 @@
         'numeric' => 'Nilai harus berisi minimal :min.',
         'string'  => 'String harus berisi minimal :min karakter.',
     ],
-    'multiple_of'          => 'The value must be a multiple of :value',
+    'multiple_of'          => 'Isian :attribute harus merupakan kelipatan dari :value.',
```
Comment (luisprmat) [inline locales/id/validation-inline.php:93]: keep in mind that the validation-inline.php file does not come with Laravel and the idea of this file is not to put a specific name to each attribute (as in validation.php) but a generic name for the validation attributes. Therefore in the translations of this file the placeholder :attribute should not appear. [View here](https://laravel-lang.github.io/lang/contributing-to-dev.html)

**Review by luisprmat — CHANGES_REQUESTED** [review]

**Comment by luisprmat** [comment]
This is the reason for the failed tests

**Comment by itsfaqih** [comment]
Sorry, I didn't see this pull request and have translated many of them (the json included). I wonder if I still could make the pull request?

File: locales/id/validation-inline.php (hunk 1 above)
Comment (andrey-helldar) [reply locales/id/validation-inline.php:93]: image

**Comment by andrey-helldar** [comment]
How to find out exactly what the problem is:

image

image

**Comment by bagoesz21** [comment]
oh, i see, sorry, i'm not focused. Thank you for reminding me.

**Pull request closed without merging**